profiles/common/modules/features/nexteuropa_editorial/nexteuropa_editorial.install

```diff
@@ -268,3 +256,77 @@ function nexteuropa_editorial_update_7002() {
   // WYSIWYG tracking settings for "Published" status case.
   variable_set('nexteuropa_editorial_tracking_status', 1);
 }
+
+/**
+ * NEPT-202: Enable nexteuropa_trackedchanges module.
+ *
+ * Update enables the module only if tracked changes are detected and
+ * migrate nexteuropa_editorial settings set here in the module to enable.
+ */
+function nexteuropa_editorial_update_7004(&$sandbox) {
```

Comment:
We should move this in a more "historically established" feature since some subsites might not have `nexteuropa_editorial` enabled. We should probably go for `cce_basic_config`